implementations/monix/src/test/scala/com/softwaremill/sttp/impl/monix/MonixTestStreamingBackend.scala

```diff
@@ -0,0 +1,22 @@
+package com.softwaremill.sttp.impl.monix
+
+import java.nio.ByteBuffer
+
+import _root_.monix.eval.Task
+import _root_.monix.reactive.Observable
+import com.softwaremill.sttp.testing.streaming.{ConvertToFuture, TestStreamingBackend}
+
+trait MonixTestStreamingBackend extends TestStreamingBackend[Task, Observable[ByteBuffer]] {
+
+  override implicit def convertToFuture: ConvertToFuture[Task] = com.softwaremill.sttp.impl.monix.convertToFuture
```

Comment:
a minor thing, but maybe `com.softwaremill.sttp.impl.monix.convertToFuture` could simply be called `convertMonixTaskToFuture` and then we wouldn't have to do the full path?

Reply:
Good idea, done